docs/install.rst

```diff
@@ -16,13 +16,17 @@ Requirements
 ############
 
 CherryPy does not have any mandatory requirements. However certain features it comes with
-will require you install certain packages.
+will require you install certain packages. To simplify installing additional
+dependencies CherryPy enables you to specify extras in your requirements (e.g.
+`cherrypy[json,routes_dispatcher,ssl]`):
```

Comment:
I think in RST, you need double backticks; single-backticks has special meaning.